keyboards/switchplate/southpaw_fullsize/southpaw_fullsize.h

```diff
@@ -0,0 +1,71 @@
+) \
+{ \
+	{ K000,  K001,  K002,  KC_NO, K004,  KC_NO, K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014,  K015,  K016,  K017,  KC_NO, K019,  K020,  K021  }, \
+	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  K114,  K115,  K116,  K117,  K118,  K119,  K120,  K121  }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  K214,  K215,  K216,  K217,  KC_NO, K219,  K220,  K221  }, \
+	{ K300,  K301,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  K314,  K315,  K316,  K317,  KC_NO, KC_NO, KC_NO, KC_NO }, \
+	{ K400,  K401,  K402,  K403,  K404,  K405,  K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413,  K414,  K415,  K416,  KC_NO, K418,  KC_NO, K420,  KC_NO }, \
+	{ K500,  K501,  K502,  K503,  K504,  K505,  K506,  KC_NO, KC_NO, KC_NO, K510,  KC_NO, KC_NO, K513,  K514,  K515,  K516,  KC_NO, KC_NO, K519,  K520,  K521  }  \
+}
+
+#define LAYOUT_ansi_wkl( \
+	K000, K001, K002,         K004,       K006, K007, K008, K009,    K010, K011, K012, K013,    K014, K015, K016, K017,   K019, K020, K021, \
+	K100, K101, K102, K103,   K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, K115, K116, K117, K118,   K119, K120, K121, \
+	K200, K201, K202, K203,   K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216,       K217,   K219, K220, K221, \
+	K300, K301, K302, K303,   K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315,             K317,                     \
+	K400, K401, K402, K403,   K404,       K406, K407, K408, K409, K410, K411, K412, K413, K414, K415,       K416, K418,         K420,       \
+	K500, K501, K502, K503,   K504, K505, K506,                       K510,                           K514, K515, K516,   K519, K520, K521  \
+) \
+{ \
+	{ K000,  K001,  K002,  KC_NO, K004,  KC_NO, K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014,  K015,  K016,  K017,  KC_NO, K019,  K020,  K021  }, \
+	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  K114,  K115,  K116,  K117,  K118,  K119,  K120,  K121  }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  K214,  K215,  K216,  K217,  KC_NO, K219,  K220,  K221  }, \
+	{ K300,  K301,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  K314,  K315,  KC_NO, K317,  KC_NO, KC_NO, KC_NO, KC_NO }, \
+	{ K400,  K401,  K402,  K403,  K404,  KC_NO, K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413,  K414,  K415,  K416,  KC_NO, K418,  KC_NO, K420,  KC_NO }, \
+	{ K500,  K501,  K502,  K503,  K504,  K505,  K506,  KC_NO, KC_NO, KC_NO, K510,  KC_NO, KC_NO, KC_NO, K514,  K515,  K516,  KC_NO, KC_NO, K519,  K520,  K521  }  \
+}
+
+#define SUPEROMEGAWHATTHEFUCK 0
```

Comment:
```suggestion
```

I don't see this used anywhere.

Reply:
Probably a leftover from when I spent hours trying to get it to compile.
Turns out QMK doesn't compile if a dash is used anywhere in the keyboard name.

Reply:
Oh yeah, dashes in keyboard names definitely won’t work.